# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-vulkan/src/info.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-vulkan/src/info.rs b/third_party/rust/gfx-backend-vulkan/src/info.rs
deleted file mode 100644
--- a/third_party/rust/gfx-backend-vulkan/src/info.rs
+++ /dev/null
@@ -1,5 +0,0 @@
-pub mod intel {
-    pub const VENDOR: u32 = 0x8086;
-    pub const DEVICE_KABY_LAKE_MASK: u32 = 0x5900;
-    pub const DEVICE_SKY_LAKE_MASK: u32 = 0x1900;
-}